views/offramp/securityQuestion.pug

```diff
@@ -0,0 +1,23 @@
+extends ../base
+include ../_includes/banner
+
+block variables
+  -var title = __('File your taxes another way')
+
+block content
+
+  h1 #{title}
+
+  .offramp
+    div
+      p #{__('You can file your taxes even if you can’t answer any security questions. Other free and secure ways to file are:')}
+      ul
+        li #{__('By mail, using the form sent with the CRA')} <i>#{__('service.name')}</i>  #{__('letter')}
```

Comment:
There's 2 spaces between service.name and "letter"

Because of whitespace formatting issues with the service name in french, can we write it like this?

`li #{__('By mail, using the form sent with the CRA')} <i>#{__('service.name')}</i>#{__(' letter')}`